lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -114,14 +122,17 @@ def save_custom_field_values
         def reset_custom_values!
           @custom_field_values = nil
           @custom_field_values_changed = true
-          values = custom_values.inject({}) { |h, v| h[v.custom_field_id] = v.value; h }
+          values = custom_values.inject({}) do |hash, custom_value|
```

Comment:
@NobodysNightmare hound is right here. Is there some side effect I am not aware of? If so, this should be noted in a comment. If not, this should be deleted. Please let it be the later.